source/js/components/RecipeCard.js

```diff
@@ -1,15 +1,34 @@
+import * as fetcherFuncs from "../storage/fetcher.js";
+import * as storageFuncs from "../storage/storage.js";
+
 class RecipeCard extends HTMLElement {
   constructor() {
     super();
-    this.attachShadow({ mode: 'open' });
+    this.attachShadow({ mode: "open" });
     this.saved = false;
+    this.created = false;
   }
 
   set data(data) {
     // TODO: update info based on json data
     this.json = data;
-
-    const styleElem = document.createElement('style');
+    // check if it's stored in local database
+    const allSavedList = fetcherFuncs.getAllSavedRecipeId();
+    let savedList = [];
+    let createdList = [];
```

Comment:
Make a GitHub Issue to note that this is hardcoded so we know where to look when we go back to implement the additional lists functionality.